roles/lighthouse/defaults/main.yaml

```diff
@@ -55,7 +55,12 @@ lighthouse_container_command:
   - --metrics-allow-origin=*
   - --metrics-port={{ lighthouse_ports_metrics }}
 lighthouse_container_command_extra_args: []
-  # - --checkpoint-sync-url=http://your-other-node
+
+checkpoint_sync_url: https://beaconstate.ethstaker.cc
+checkpoint_sync_enabled: false
```

Comment:
I think we could do the following, so that it follows +/- the existing convention:

1. Remove the `checkpoint_sync_enabled` variable from the defaults.

2. Just leave the lighthouse specific one.
```yaml
lighthouse_checkpoint_sync_enabled: false
```

3. Add `ethereum_node_cl_checkpoint_sync_enabled: false` to `roles/ethereum_node/defaults/main.yaml` 

4. Inject the var in `roles/ethereum_node/tasks/setup_cl.yaml` 

E.g. 

```yaml
- name: "Consensus client: lighthouse"
  when: ethereum_node_cl == "lighthouse"
  ansible.builtin.include_role:
    name: ethpandaops.general.lighthouse
  vars:
    lighthouse_announced_ip: "{{ ethereum_node_announced_ip }}"
    lighthouse_ports_p2p_tcp: "{{ ethereum_node_cl_ports_p2p_tcp }}"
    ...
    lighthouse_checkpoint_sync_enabled: "{{ ethereum_node_cl_checkpoint_sync_enabled }}"
  ```
  
  Repeat for the other roles.

Reply:
Fixed based on feedback.